config/render/capm3.yaml

```diff
@@ -2412,17 +2413,18 @@ subjects:
   namespace: baremetal-operator-system
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
+kind: RoleBinding
 metadata:
   name: baremetal-operator-manager-rolebinding
+  namespace: single-ns-bmh
 roleRef:
   apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: baremetal-operator-manager-role
```

Comment:
This should not change and same with serviceAccount, I will check on this

Reply:
It was the patch that was overwriting the name-prefix, had to do trickery through another patch to keep the prefix....
Of course this is not ideal as it is set static, if others have ideas they are welcome